pig/src/main/java/org/jboss/pnc/bacon/pig/impl/nvr/NvrListGenerator.java

```diff
@@ -38,11 +39,11 @@
     private NvrListGenerator() {
     }
 
-    public static boolean generateNvrList(Map<String, Collection<String>> checksums, String targetPath) {
+    public static boolean generateNvrList(Map<String, Collection<String>> checksums, Path targetPath) {
         log.info("Generating NVR list for {} checksums and saving result to {}", checksums.size(), targetPath);
 
         List<KojiBuild> builds = BuildFinderUtils.findBuilds(checksums, true);
-        File outputDirectory = new File(FilenameUtils.getPath(targetPath));
+        File outputDirectory = targetPath.toFile().getParentFile();
```

Comment:
Might be better to get the parent and then do the conversion `targetPath.getParent().toFile()`?

Reply:
do you think there is any real difference between the two?

Reply:
My original idea was just to use the new `Path` API and save the conversion to `File` to the last moment.

With `Path.getParent()`, it is going to give weird behavior if the path is not `normalize()`d first. I am not sure if the `File` API does the same thing. For example, I think the parent of `parent/dir/../` is `parent/dir` and not `parent` like you would want.